### PR TITLE
Added Hammond to the Projects section.

### PR DESCRIPTION
**Pull request by alatiera**
I've written a podcast client while playing with the bindings.

Ps. The docs are awesome!

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -108,5 +108,6 @@ features = ["v3_10"]
 * [PNMixer-rs](https://github.com/hasufell/pnmixer-rust)
 * [BrewStillery](https://github.com/MonkeyLog/BrewStillery)
 * [Way Cooler](https://github.com/way-cooler)
+* [Hammond}(https://gitlab.gnome.org/alatiera/Hammond)
```
Comment (GuillaumeGomez) [inline index.md]: This isn't valid markdown. ;)

Comment (alatiera) [reply index.md]: Damned typo, should be fixed now

**Comment by GuillaumeGomez** [comment]
Please [squash](https://blog.guillaume-gomez.fr/Git-tips/1/2) now. :)

**Comment by alatiera** [comment]
neat trick! thanks for documenting it.

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**